week-12/homework-1.1.html

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Get to know Bootstrap</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css"
+    />
+  </head>
+
+  <body>
+    <div class="navbar sticky-top bg-dark flex-md-nowrap p-0 shadow">
+      <a
+        class="navbar-brand col-md-3 col-lg-2 me-0 px-3 fs-3 text-white"
+        href="#"
+        >My app</a
+      >
+      <button
+        class="nav-link px-3 d-md-none text-white"
+        type="button"
+        data-bs-toggle="offcanvas"
+        data-bs-target="#menu"
+      >
+        <i class="bi bi-list me-1"></i>
+        Menu
+      </button>
+    </div>
+    <div class="container-fluid">
+      <div class="row">
+        <div
+          class="sidebar border border-right col-md-3 col-lg-2 p-0 bg-body-tertiary"
+        >
+          <div id="menu" class="offcanvas-md offcanvas-end bg-body-tertiary">
+            <div class="offcanvas-header">
+              <h5 class="offcanvas-title">Menu</h5>
+              <button
+                type="button"
+                class="btn-close"
+                data-bs-dismiss="offcanvas"
+                data-bs-target="#menu"
+              ></button>
+            </div>
+            <div class="offcanvas-body">
+              <ul class="nav flex-column">
+                <li class="nav-item">
+                  <a
+                    class="nav-link d-flex align-items-center gap-2 active"
+                    aria-current="page"
+                    href="#"
+                  >
+                    <i class="bi bi-speedometer2"></i>
+                    Dashboard
+                  </a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link d-flex align-items-center gap-2" href="#">
+                    <i class="bi bi-file-earmark"></i>
+                    Orders
+                  </a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link d-flex align-items-center gap-2" href="#">
+                    <i class="bi bi-cart"></i>
+                    Products
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+
+        <div class="col-md-9 col-lg-10 p-4">
+          <div id="carouselExample" class="carousel slide">
+            <div class="carousel-indicators">
+              <button
+                type="button"
+                data-bs-target="#carouselExample"
```

Comment:
If we change our id above, we need to change it here to match the id above as well, change it to `data-bs-target="#promo-carousel"`